engine/src/main/scala/cromwell/engine/backend/JobDescriptor.scala

```diff
@@ -17,7 +17,7 @@ import scala.util.Try
   *
   * @tparam K CallKey subtype
   */
-sealed trait JobDescriptor[K <: JobKey] {
+sealed trait JobDescriptor[K <: CallKey] {
```

Comment:
Yes, the LSF backend is based off of SGE backend.

I have not checked out the "Develop" branch yet.